src/user_management/user.model.js

```diff
@@ -1,11 +1,37 @@
+import bcrypt from 'bcrypt'
+
 export default (sequelize, DataTypes) => {
   const User = sequelize.define('user', {
     id: {
       type: DataTypes.UUID,
       primaryKey: true,
       defaultValue: DataTypes.UUIDV4
+    },
+    firstName: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    lastName: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    email: {
+      type: DataTypes.STRING,
+      unique: true,
+      allowNull: false
+    },
+    password: {
+      type: DataTypes.STRING,
+      allowNull: false
     }
-  })
+  }, { timestamps: false })
 
+  User.Prototype.generateHash = function (password) {
+    return bcrypt.hashSync(password, bcrypt.genSaltSync(9))
```

Comment:
Don't perform synchronous operations. The server will stop handling requests while the operation is running. Always use asynchronous alternatives when available. This library in particular has asynchronous alternatives that use promises.